colorscad.sh

```diff
@@ -18,6 +19,7 @@ Options
   -j  Maximum number of parallel jobs to use: defaults to 8, reduce if you're low on RAM
   -o  Output file: it must not yet exist (unless option -f is used),
       and must have as extension either '.amf' or '.3mf'
+  -p  The path to the openscad binary to use
```

Comment:
Let's also mention this in the README, as alternative to the PATH shenanigans mentioned there.